flask_app/__init__.py

```diff
@@ -6,6 +6,17 @@
 
 dictConfig(LOGGING_CONFIG)
 
+logging.getLogger().setLevel(0)  # Allows all messages to go through logger
+
+logging.addLevelName(9, "AddINFO")  # Create a new level below DEBUG
+
+
+def AddINFO(self, message, *args, **kws):
+    if self.isEnabledFor(9):
+        self._log(9, message, args, **kws)
+
+logging.Logger.AddINFO = AddINFO
```

Comment:
why do we need a new level? We can just use the debug level for logging

Reply:
I was thinking that this level could be for the information that takes up alot of space and is generally unused, like the base64 representation of images! I will remove it because of the changes to instance logging.